src/main/kotlin/com/devfloor/untitled/article/application/ArticleService.kt

```diff
@@ -14,4 +14,9 @@ class ArticleService(
     @Transactional(readOnly = true)
     fun showById(id: Long): Article = repository.findByIdOrNull(id)
         ?: throw EntityNotFoundException("사용자가 요청한 리소스가 없습니다")
+
+    @Transactional
+    fun create(article: Article): Article {
+        return repository.save(article)
+    }
```

Comment:
repository의 메서드 내부적으로 트랜잭션 처리를 한다고 합니다 ! 따라서 서비스의 한 메소드 내에서 repository에 단 한번만 접근을 한다면 @transactional 어노테이션이 불필요 할 것 같아요 !

Ref) https://github.com/dev-floor/untitled-server/pull/20#discussion_r628177767
```suggestion

    fun create(article: Article): Article {
        return repository.save(article)
    }
```